src/Components/Patient/PatientRegister.tsx

```diff
@@ -1313,6 +1324,12 @@ export const PatientRegister = (props: PatientRegisterProps) => {
                               label="Emergency contact number"
                               required
                               types={["mobile", "landline"]}
+                              disabled={isEmergencyNumberEnabled}
+                              value={
+                                isEmergencyNumberEnabled
+                                  ? field("phone_number").value
+                                  : initForm.emergency_phone_number
+                              }
```

Comment:
This will cause issues as the displayed value may not be the one that is sent in the POST request.

Update form state for emergency phone number instead of displaying a different value conditionally.